packages/app/src/systems/Bridge/components/BridgeTxItem/BridgeTxItem.tsx

```diff
@@ -60,14 +57,30 @@ export const BridgeTxItem = ({
 
 const styles = cssObj({
   listItem: cssObj({
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    px: '0px !important',
-    py: '$1 !important',
-    width: '$full',
+    flexDirection: 'column',
+    border: '1px solid $intentsBase6 !important',
+    backgroundColor: '$intentsBase0',
+    width: '344px',
```

Comment:
the idea is that the item will get `$full`, and parent List element will choose width